### PR TITLE
Add filter for languages with ll_CC description

### PR DESCRIPTION
**Pull request by columbarius**
Implementing the proposed solution in #125

### CODE REVIEW

File: tldr.py [hunk 1]
```diff
@@ -30,10 +30,19 @@
...
+    elif lang == "pt":
+        return "pt_PT"
+    else:
+        return lang.split('_')[0]
+
+DEFAULT_LANG = filter_languages(
+    os.environ.get(
+        'LANG',
+        'C'
+    ).split('.')[0])
```
Comment (columbarius) [inline tldr.py:45]: Does anyone know how to apply this filter with chain notation for a nicer syntax like
`.split('.')[0].apply(filter_languages)`? 
Couldn't find a solution with a quick search.

**Comment by MasterOdin** [comment]
Thanks for this! Sorry for taking a bit to review, some minor comments, but definitely an improvement on the state of things!

File: tldr.py (hunk 1 above)
Comment (columbarius) [reply tldr.py]: Sure, that makes sense.

File: tldr.py [hunk 2]
```diff
@@ -30,10 +30,20 @@
     'https://tldr-pages.github.io/assets/tldr.zip'
 )
 
-DEFAULT_LANG = os.environ.get(
-    'LANG',
-    'C'
-).split('_')[0]
+
+def filter_languages(lang):
```
[the remark this answers is not in the document]
Comment (columbarius) [reply tldr.py]: Ok, for me a filter is something which modifies or removes an input and outputs the result, but I see your point.

**Comment by columbarius** [comment]
No Problem. I noticed, open source can take it's time, since it's freetime for most of us. Thanks for the review.

**Comment by columbarius** [comment]
Thanks for merging!

**Pull request merged**